.idea/codeStyles/Project.xml

```diff
@@ -47,6 +46,8 @@
           <emptyLine />
         </value>
       </option>
+      <option name="ALIGN_MULTILINE_RECORDS" value="false" />
```

Comment:
I took the liberty to set the margin lines and to change alignment according to my personal preferences since we have no style guide... I'm happy to undo this though.